aggregator/generic_elem.go

```diff
@@ -347,7 +348,7 @@ func (e *GenericElem) Consume(
 		// We only attempt to consume if the outgoing metrics type is local instead of forwarded.
```

Comment:
Do you need to update the comments here?

Reply:
Sure can do.